src/scss/sections/_howto.scss

```diff
@@ -24,34 +24,36 @@
 	.carousel-control-prev {
 		&:before,
 		.sni {
-			margin-top: -7em;
-			margin-right: -2em;
+			top: 4.5em;
+			left: 1em;
 
 			@include media-breakpoint-up(lg) {
-				margin-right: -7em;
-				margin-top: -4.5em;
+				left: 6em;
+				top: 12em;
 			}
 		}
 	}
 
 	.carousel-control-next {
 		&:before,
 		.sni {
-			margin-top: -7em;
-			margin-left: -2em;
+			top: 4.5em;
+			right: 1em;
 
 			@include media-breakpoint-up(lg) {
-				margin-left: -7em;
-				margin-top: -4.5em;
+				right: 6em;
+				top: 12em;
```

Comment:
this solution is not responsive

Reply:
I didn't find the solution, it seems we need to detect the image height to keep the arrow in the center of the images, can't find a way with css, unless we use JS
I am using extra media queries to adjust the arrow position for this solution for now